memegen/routes/_common.py

```diff
@@ -31,11 +32,36 @@ def samples(app):
         }
 
 
-def track(app, request, title):
-    """Log the requested content."""
+def display(title, path, mimetype='image/jpeg'):
+    """Render a webpage or raw image based on request."""
+    mimetypes = request.headers.get('Accept', "").split(',')
+    browser = 'text/html' in mimetypes
```

Comment:
Thanks for the info! I thought when I was playing around with it `'text/html' in request.accept_mimetypes` was `True` even for `*/*`, but I might have missed something.